modules/utilityFunction.js

```diff
@@ -0,0 +1,127 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
```

Comment:
- [ ] Please don't disable linters as linters are there to insure best practices and prevent bad code from being committed to the default branch. I would suggest you to manually fix the linter errors instead of disabling them.